src/components/LeaderboardCard/LeaderboardCard.scss

```diff
@@ -0,0 +1,103 @@
+@use "../../styles/reset";
+@use "../../styles/variables" as *;
+@use "../../styles/typography";
+
+.leaderboard-card {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 80%;
+  margin: 1rem auto;
+  border: 1px $light-grey-color solid;
+  border-radius: 10px;
+  padding: 0.3rem;
+  &__img-div {
+    width: 1.8rem;
```

Comment:
I think that the image needs to be larger at the moment it looks too small. 
It would be worth adding some more padding to the sides of the `.leaderboard-card` as well. 

<img width="436" alt="image" src="https://github.com/nology-tech/wov-client-project/assets/55165280/5c317744-f5b5-44f0-b045-5f0139f61a0b">